packages/blade/src/components/Input/BaseInput/BaseInputAnimatedBorder.native.tsx

```diff
@@ -42,14 +48,19 @@ export const BaseInputAnimatedBorder = ({
   });
 
   useEffect(() => {
-    if (currentInteraction == 'focus') {
+    console.log({ currentInteraction, validationState });
```

Comment:
Remove log

Reply:
fixed in 717f7a1da44ddbaf76edc37e72d60e182d95576b